CI: test_export_create_help が Rich ANSI エスケープで失敗

## 概要

\`tests/unit/cli/test_commands_export.py::test_export_create_help\` がCI Unit Testsで失敗。Typer/Rich が help 出力に挿入する ANSI エスケープシーケンスにより、\`--project\` 文字列が分割されて \`in\` 判定を通らない。

## 失敗ログ

\`\`\`
AssertionError: assert '--project' in '...\x1b[1;36m-\x1b[0m\x1b[1;36m-project\x1b[0m...'
\`\`\`

(CI run: https://github.com/NEXTAltair/LoRAIro/actions/runs/24323174118)

## 原因

CI環境では Rich がターミナルを検出する条件が異なり、ローカル実行時と異なる ANSI 出力になる。テストは \`--project\` がそのまま出力される前提で assert しているため失敗する。

## 影響箇所

\`tests/unit/cli/test_commands_export.py:331-339\`

\`\`\`python
def test_export_create_help() -> None:
    result = runner.invoke(app, ["export", "create", "--help"])
    assert result.exit_code == 0
    assert "--project" in result.stdout
    assert "--output" in result.stdout
    assert "--format" in result.stdout
    assert "--resolution" in result.stdout
\`\`\`

同様のパターンが他の \`test_*_help\` テストにも存在する可能性あり。

## 修正案

### Option A: カラー無効化
\`\`\`python
import os
result = runner.invoke(app, ["export", "create", "--help"], env={"NO_COLOR": "1", "TERM": "dumb"})
\`\`\`

### Option B: ANSI 除去してから assert
\`\`\`python
import re
def strip_ansi(s: str) -> str:
    return re.sub(r"\x1b\[[0-9;]*m", "", s)

clean = strip_ansi(result.stdout)
assert "--project" in clean
\`\`\`

### Option C: CliRunner 設定
\`CliRunner(mix_stderr=False)\` や Typer 側で color を disable。

## 推奨

Option A (NO_COLOR) が最小変更でわかりやすい。conftest.py に共通フィクスチャ化する手もある。

## 優先度

**高** — CIの Unit Tests ジョブで \`-x\` オプションにより全テストが中断される。

--- a/tests/unit/cli/conftest.py
+++ b/tests/unit/cli/conftest.py
@@ -1,0 +1,20 @@
+"""CLI テスト共通設定。
+
+Rich/Typer のカラー出力を無効化する autouse フィクスチャを提供する。
+"""
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def disable_rich_color(monkeypatch: pytest.MonkeyPatch) -> None:
+    """Rich/Typer のカラー出力をテスト環境で無効化する。
+
+    Rich は help 出力中の `--flag` パターンに ANSI エスケープ (`\\x1b[1;36m`) を挿入するため、
+    CI 環境のようにカラーが強制される条件下では `assert "--project" in result.stdout` が
+    `\\x1b[1;36m-\\x1b[0m\\x1b[1;36m-project\\x1b[0m` との比較になり失敗する。
+
+    NO_COLOR と TERM=dumb を設定して Rich の自動検出を抑制する。
+    """
+    monkeypatch.setenv("NO_COLOR", "1")
+    monkeypatch.setenv("TERM", "dumb")

